django-stubs/contrib/gis/db/models/fields.pyi

```diff
@@ -23,18 +23,66 @@ _ST = TypeVar("_ST")
 # __get__ return type
 _GT = TypeVar("_GT")
 
+_ST_PointField = TypeVar("_ST_PointField", default=Point | Combinable)
+_GT_PointField = TypeVar("_GT_PointField", default=Point)
+_Form_PointField = TypeVar("_Form_PointField", bound=forms.PointField, default=forms.PointField)
+_Geom_PointField = TypeVar("_Geom_PointField", bound=Point, default=Point)
+
+_ST_LineStringField = TypeVar("_ST_LineStringField", default=LineString | Combinable)
+_GT_LineStringField = TypeVar("_GT_LineStringField", default=LineString)
+_Form_LineStringField = TypeVar("_Form_LineStringField", bound=forms.LineStringField, default=forms.LineStringField)
+_Geom_LineStringField = TypeVar("_Geom_LineStringField", bound=LineString, default=LineString)
+
+_ST_PolygonField = TypeVar("_ST_PolygonField", default=Polygon | Combinable)
+_GT_PolygonField = TypeVar("_GT_PolygonField", default=Polygon)
+_Form_PolygonField = TypeVar("_Form_PolygonField", bound=forms.PolygonField, default=forms.PolygonField)
+_Geom_PolygonField = TypeVar("_Geom_PolygonField", bound=Polygon, default=Polygon)
+
+_ST_MultiPointField = TypeVar("_ST_MultiPointField", default=MultiPoint | Combinable)
+_GT_MultiPointField = TypeVar("_GT_MultiPointField", default=MultiPoint)
+_Form_MultiPointField = TypeVar("_Form_MultiPointField", bound=forms.MultiPointField, default=forms.MultiPointField)
+_Geom_MultiPointField = TypeVar("_Geom_MultiPointField", bound=MultiPoint, default=MultiPoint)
+
+_ST_MultiLineStringField = TypeVar("_ST_MultiLineStringField", default=MultiLineString | Combinable)
+_GT_MultiLineStringField = TypeVar("_GT_MultiLineStringField", default=MultiLineString)
+_Form_MultiLineStringField = TypeVar(
+    "_Form_MultiLineStringField", bound=forms.MultiLineStringField, default=forms.MultiLineStringField
+)
+_Geom_MultiLineStringField = TypeVar("_Geom_MultiLineStringField", bound=MultiLineString, default=MultiLineString)
+
+_ST_MultiPolygonField = TypeVar("_ST_MultiPolygonField", default=MultiPolygon | Combinable)
+_GT_MultiPolygonField = TypeVar("_GT_MultiPolygonField", default=MultiPolygon)
+_Form_MultiPolygonField = TypeVar(
+    "_Form_MultiPolygonField", bound=forms.MultiPolygonField, default=forms.MultiPolygonField
+)
+_Geom_MultiPolygonField = TypeVar("_Geom_MultiPolygonField", bound=MultiPolygon, default=MultiPolygon)
+
+_ST_GeometryCollectionField = TypeVar("_ST_GeometryCollectionField", default=GeometryCollection | Combinable)
```

Comment:
I propose moving these variables closer to the class that use them. It would be easier to review.